prog/postgres/postgres_server_nexus.rb

```diff
@@ -18,6 +18,11 @@ def self.assemble(resource_id:, timeline_id:, timeline_access:, representative_a
       ubid = PostgresServer.generate_ubid
 
       postgres_resource = PostgresResource[resource_id]
+      boot_image = case postgres_resource.flavor
+      when PostgresResource::Flavor::PARADEDB then "postgres-paradedb-ubuntu-2204"
+      else "postgres-ubuntu-2204"
```

Comment:
We could add `PostgresResource::Flavor::POSTGRES` as a case, and fails in the `else` branch. This approach helps guard against unwanted flavors, especially when we deprecate some. Otherwise, it will silently use the default one.

Reply:
Makes sense, added additional check